packages/xo-server/src/xapi/mixins/patching.mjs

```diff
@@ -533,14 +533,14 @@ const methods = {
         beforeEvacuateVms: async () => {
           // On XS/CH, start by installing patches on all hosts
           if (!isXcp) {
-            Task.run({ properties: { name: `Installing XS patches` } }, async () => {
+            return Task.run({ properties: { name: `Installing XS patches` } }, async () => {
```

Comment:
The real problem is that the installing patches is not correctly awaited before evacuating VMs.

Maybe this should ship in a patch?

/cc @fbeauchamp

Reply:
You're right, I was focusing on solving the problem I had in mind, I did not realize the bug was actually way more problematic. I'm editing the changelog entry.
I also think it should go in a patch

Reply:
I think this could be a patch

Reply:
IMHO, it should, RPU does not work as it is.